### DISCUSSION TITLE
Could I align my multi line input code afer Insertion Operators?

### DISCUSSION THREAD

**Discussion by wy-luke** [microsoft/vscode-cpptools#10017] [Q&A]
I want my code be aligned like this:

```c++
    Eigen::Matrix4f translate;
    translate << 1,0,0,-eye_pos[0],
                 0,1,0,-eye_pos[1],
                 0,0,1,-eye_pos[2],
                 0,0,0,1;
```

But I almost have tried all clang-format optins, they all couldn't work as I wanted.

My code will be aligned like this now, with four spaces before all new line:
```c++
    Eigen::Matrix4f translate;
    translate << 1, 0, 0, -eye_pos[0],
        0, 1, 0, -eye_pos[1],
        0, 0, 1, -eye_pos[2],
        0, 0, 0, 1;
```

I'm using the latest vscode and cpp extention. Thanks!!!!

**Comment by sean-mcmanus** [comment]
I don't know.

**Reply by wy-luke** [reply]
😂😂Thank you anyway.

**Reply by michelleangela** [reply]
Perhaps https://github.com/llvm/llvm-project/issues might have users who might know more about certain [clang-format style options](https://clang.llvm.org/docs/ClangFormatStyleOptions.html)?

**Reply by wy-luke** [reply]
> Perhaps https://github.com/llvm/llvm-project/issues might have users who might know more about certain [clang-format style options](https://clang.llvm.org/docs/ClangFormatStyleOptions.html)?

yes，you are right! Thank you!
